examples/downstream_requests.rs

```diff
@@ -22,14 +23,9 @@ fn main() {
         graph.merge_frame(frame_graph, &remote_frame_id);
     });
 
-    let mut edge_id: EdgeId = EdgeId::from(id);
-    if let Some(frame_id) = frame_id {
-        edge_id = edge_id.copy_for_frame_id(&frame_id)
-    }
-
     let downstream_effects = graph.all_downstream_effects_of(graph.edges.get(&edge_id).unwrap()).into_iter().filter_map(|edge| {
         if let EdgeType::RequestStart { request_id, .. } = edge.edge_type {
-            Some((format!("{:?}", edge.id), request_id))
+            Some((request_id, format!("{}", edge.id)))
```

Comment:
@ShivanKaul I've also snuck in a change here to reverse the order of returned arguments here, to make it consistent across both steps.

Reply:
Ha I made this same change locally!

Reply:
unrelated: I think we're missing the `clap` dependency in `Cargo.toml` - I had to add that locally.